src/utils/Chart.js

```diff
@@ -0,0 +1,42 @@
+const DataLoader = require("./DataLoader");
+
```

Comment:
Chart class should either contain PlotData class to store all data points and timestamps to show on the chart or replace it entirely